### PR TITLE
Update show_data so that 'class_map' can be passed to show_preds --> see #811

### PR DESCRIPTION
**Pull request by elmourr**
this update allow to pass class_map to show_pred and to display the predictions with bounding boxes and string labels:
show_preds( preds=preds, class_map = class_map,denormalize_fn=denormalize_imagenet, ncols=3,)

### CODE REVIEW

**Review by elmourr — COMMENTED** [review]
please ignore this commit

File: icevision/visualize/show_data.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def show_record(
 
 def show_pred(
     pred: Prediction,
+    class_map: Optional[ClassMap] = None,
```
Comment (lgvaz) [inline icevision/visualize/show_data.py:70]: Heh, so the `class_map` traces on all show functions are actually a remnant of the latest refactor - they are not required anymore, the class_map is always inferred from the record, actually what we should do is remove this parameter from the functions haha

File: icevision/visualize/show_data.py [hunk 2]
```diff
@@ -83,6 +84,7 @@ def show_pred(
         display_bbox=display_bbox,
         display_mask=display_mask,
         denormalize_fn=denormalize_fn,
+        **draw_sample_kwargs,
```
Comment (lgvaz) [inline icevision/visualize/show_data.py:87]: this is a good catch yeah, once you remove the `class_map` parameter we merge this one =)

File: icevision/visualize/show_data.py (hunk 1 above)
Comment (elmourr) [reply icevision/visualize/show_data.py:70]: ahh ok - good to know, would make sense too. good catch

File: icevision/visualize/show_data.py (hunk 2 above)
Comment (elmourr) [reply icevision/visualize/show_data.py:87]: ok will do

**Review by elmourr — COMMENTED** [review]
done - Removed :+1: 
class_map: Optional[ClassMap] = None,

**Review by lgvaz — COMMENTED** [review]
Alright!! Seems good!

Now, the only thing before we merge, which I just realized, is that this PR is going to merge into the branch `parser_multiprocessing`, see it here:
image

You should change it to merge into master. I think the easiest way to do this, since it's a small change, is just to open another PR and be sure to select "into master" on the github UI

**Pull request closed without merging**